test/go-tests/test_gitcommitid.go

```diff
@@ -0,0 +1,255 @@
+package go_tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/go-utils/pkg/common/strutils"
+	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+	"github.com/stretchr/testify/require"
+)
+
+const commitIDShipyard = `--- 
+apiVersion: spec.keptn.sh/0.2.0
+kind: Shipyard
+metadata: 
+  name: shipyard-quality-gates
+spec: 
+  stages: 
+    - 
+      name: hardening`
+
+const gitCommitIDSLO = `---
+spec_version: "0.1.1"
+comparison:
+  aggregate_function: "avg"
+  compare_with: "single_result"
+  include_result_with_score: "pass"
+  number_of_comparison_results: 1
+filter:
+objectives:
+  - sli: "response_time_p95"
+    key_sli: false
+    pass:             # pass if (relative change <= 75% AND absolute value is < 75ms)
+      - criteria:
+          - "<=+75%"  # relative values require a prefixed sign (plus or minus)
+          - "<800"     # absolute values only require a logical operator
+    warning:          # if the response time is below 200ms, the result should be a warning
+      - criteria:
+          - "<=1000"
+          - "<=+100%"
+    weight: 1
+  - sli: "throughput"
+    pass:
+      - criteria:
+          - "<=+100%"
+          - ">=-80%"
+  - sli: "error_rate"
+total_score:
+  pass: "90%"
+  warning: "75%"`
+
+func Test_EvaluationGitCommitID(t *testing.T) {
+	projectName := "commit-id-evaluation"
+	serviceName := "my-service"
+	shipyardFilePath, err := CreateTmpShipyardFile(commitIDShipyard)
+	require.Nil(t, err)
+	defer os.Remove(shipyardFilePath)
+
+	t.Log("deleting lighthouse configmap from previous test run")
+	_, _ = ExecuteCommand(fmt.Sprintf("kubectl delete configmap -n %s lighthouse-config-keptn-%s", GetKeptnNameSpaceFromEnv(), projectName))
```

Comment:
pls use `ExecuteCommandf` here and in other places... and pls handle err

Reply:
Actually only in this place, we can receive an error that the configmap does not exists, it will return an error, as it cannot delete it, if it exists, it just deletes it

Reply:
If you don't want to handle the error, then leave out the assignment to the blank identifier (`_`)

Reply:
fixed